src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -61,6 +61,14 @@
 from Utils.PortForward import portForward, PortForward
 
 
+def get_token(env=None):
```

Comment:
Valentin, I believe at some point we will want to have different operations for tokens (like, renewing, checking validity, etc) and maybe even different technologies. Right?

In that case, I would suggest to create a new module under the `Utils` package and have all the token-based implementation in there.